front/app/api/app_configuration/types.ts

```diff
@@ -157,6 +140,10 @@ export interface IAppConfigurationSettings {
     allowed: boolean;
     enabled: boolean;
   };
+  vienna_citizen_login?: {
+    allowed: boolean;
+    enabled: boolean;
+  };
```

Comment:
This was first defined in the vienna auth module. That module has been removed, so now it's here.